packages/fast-component-explorer/app/explorer.style.ts

```diff
@@ -1,8 +1,11 @@
-import { ComponentStyles, ManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { ComponentStyles } from "@microsoft/fast-jss-manager-react";
 import { DesignSystem } from "@microsoft/fast-components-styles-msft";
 
 export interface ExplorerClassNameContract {
     explorer?: string;
+    explorer_controls?: string;
+    explorer_viewerContrainer?: string;
```

Comment:
contrainer? even without the spelling issue, I would like this to be `explorer_viewerRegion`